dataservice/util/data_gen/data_generator.py

```diff
@@ -0,0 +1,183 @@
+from datetime import datetime
+from dateutil import relativedelta
+import uuid
+import random
+
+from dataservice.util.data_gen.data import *
```

Comment:
`*` should be avoided.
You can look up python start imports for all the reasons it's bad.